### PR TITLE
Mailinglist UI improvements

### PR DESCRIPTION
**Pull request by giplt**
A number of small changes to improve the workflow of sending out a message to a mailing list

### CODE REVIEW

File: system/datatypes/mailinglist.php [hunk 1]
```diff
@@ -229,7 +230,7 @@ private function displayArchive() {
 				$date = '';
 			}
 			$row->addCell($date);
-			$buttons = $this->makeActionButton(__('read'), $mailing->getId());
+			$buttons = $this->makeActionButton(__('view'), $mailing->getId());
```
Comment (matthijskooijman) [inline system/datatypes/mailinglist.php]: This change does not seem to be mentioned in the commit message? I suspect it would be a separate commit even?

File: system/datatypes/mailinglist.php [hunk 2]
```diff
@@ -537,6 +539,21 @@ private function showMailingAction($mailingId) {
...
+			} catch (\Exception $e) {
+				$msg = $e->getMessage();
+				$msgType = 'error';
+			}
+			
+			// goto view page with notification
+			notify($msgType, $msg);
+			return ['redirect', $this->constructFullPath($this->pagename . '/' . $mailingId)];
+		}
+
```
Comment (matthijskooijman) [inline system/datatypes/mailinglist.php]: There is quite some duplication between the test and regular send case. Perhaps these could be unified a bit more?

File: system/datatypes/mailinglist.php [hunk 3]
```diff
@@ -770,6 +782,22 @@ private function createOrUpdateMailing(array $data) {
...
+		$senderName = $this->getDoc()->getAttribute('sender-name');
+		$senderEmail = $this->getDoc()->getAttribute('sender-email');
+		
+		/** @var HyphaDomElement $mailing */
+		$mailing = $this->xml->document()->getElementById($mailingId);
+		
+		$message = '<p><a href="' . $linkToMailing . '">' . __('ml-if-unreadable-use-link') . '</a></p>';
+		$message .= $mailing->getHtml();
+		$this->send($mailing->getAttribute('subject'), $message, [$email], $senderEmail, $senderName);
```
Comment (matthijskooijman) [inline system/datatypes/mailinglist.php]: Same here: This code is duplicated between test send and regular send, so it would be very easy now to update one and not the other, making the test send non-representative. I can imagine merging these two functions (with a test boolean argument), or perhaps moving the duplicate code that generates (and perhaps sends) each message into its own function.

File: system/languages/en.php [hunk 4]
```diff
@@ -238,13 +238,15 @@
...
+		"ml-test-send" => "test-send",
 		"ml-view-addresses" => "view addresses",
 		"ml-date-format" => "d/m/Y",
 		"ml-received-by" => "received by",
+		"ml-sure-to-send" => "Are you sure to send off this mailing to [[count]] recipient(s)?",
 
 		// mailing list - response
 		"ml-no-sender" => "no sender, please edit the mailing list",
-		"ml-mail-has-invalid-status" => "de mail heeft een onjuiste status",
+		"ml-mail-has-invalid-status" => "the mail has an invalid status",
```
Comment (matthijskooijman) [inline system/languages/en.php]: This also seems a separate change.

File: system/languages/nl.php [hunk 5]
```diff
@@ -154,7 +154,7 @@
 		"general-info" => "algemene informatie",
 		"lineup" => "lineup",
 		"timetable" => "blokkenschema",
-		"view" => "view",
+		"view" => "bekijk",
```
Comment (matthijskooijman) [inline system/languages/nl.php]: This one as well (can probably be in a commit together with the other translation fix above)

**Comment by giplt** [comment]
Just split this PR into 4 separate ones: #165, #166, #167, #168.
